grid2op/Environment/MultiEnv.py

```diff
@@ -11,383 +11,43 @@
-
-        Returns
-        -------
-        res: ``list``
-            The list of all observations. This list counts :attr:`MultiEnvironment.nb_env` elements, each one being
-            an :class:`grid2op.Observation.BaseObservation`.
-
-        """
-        for remote in self._remotes:
-            remote.send(('r', None))
-        res = [self.imported_env.observation_space.from_vect(remote.recv()) for remote in self._remotes]
-        return np.stack(res)
-
-    def close(self):
-        """
-        Close all the environments and all the processes.
-        """
-        for remote in self._remotes:
-            remote.send(('c', None))
-
-    def set_chunk_size(self, new_chunk_size):
-        """
-        Dynamically adapt the amount of data read from the hard drive. Usefull to set it to a low integer value (eg 10
-        or 100) at the beginning of the learning process, when agent fails pretty quickly.
-
-        This takes effect only after a reset has been performed.
-
-        Parameters
-        ----------
-        new_chunk_size: ``int``
-            The new chunk size (positive integer)
-
-        """
-        try:
-            new_chunk_size = int(new_chunk_size)
-        except Exception as e:
-            raise Grid2OpException("Impossible to set the chunk size. It should be convertible a integer, and not"
-                                   "{}".format(new_chunk_size))
-
-        if new_chunk_size <= 0:
-            raise Grid2OpException("Impossible to read less than 1 data at a time. Please make sure \"new_chunk_size\""
-                                   "is a positive integer.")
-
-        for remote in self._remotes:
-            remote.send(('z', new_chunk_size))
-
-    def set_ff(self, ff_max=7*24*60/5):
-        """
-        This method is primarily used for training.
-
-        The problem this method aims at solving is the following: most of grid2op environments starts a Monday at
-        00:00. This method will "fast forward" an environment for a random number of timestep between 0 and ``ff_max``
-        """
-        try:
-            ff_max = int(ff_max)
-        except:
-            raise RuntimeError("ff_max parameters should be convertible to an integer.")
-
-        for remote in self._remotes:
-            remote.send(('f', ff_max))
-
-    def get_seeds(self):
-        """
-        Get the seeds used to initialize each sub environments.
-        """
-        for remote in self._remotes:
-            remote.send(('seed', None))
-        res = [remote.recv() for remote in self._remotes]
-        return np.stack(res)
-
-    def get_parameters(self):
-        """
-        Get the parameters of each sub environments
-        """
-        for remote in self._remotes:
-            remote.send(('params', None))
-        res = [remote.recv() for remote in self._remotes]
-        return res
-
+    def __init__(self, env, nb_env):
+        envs = [env for _ in range(nb_env)]
```

Comment:
Are we sure it's working as intended ? Or do we need to perform copy of the env argument ?

Reply:
Yes